manifests/Neotys/NeoLoad/7.3.yaml

```diff
@@ -0,0 +1,19 @@
+Id: Neotys.NeoLoad
+Name: NeoLoad
+AppMoniker: neoloadgit
```

Comment:
Please make this field consistent in both files?

Reply:
@samiranshah I have edited 7.3 and submitted a PR.

Reply:
There was a typo in the AppMoniker in 7.3.